redcap_bridge/test_redcap/test_server_elab_interface.py

```diff
@@ -0,0 +1,31 @@
+from redcap_bridge.server_elab_interface import (create_template_with_converted_csv, download_experiment,
+                                                 create_template)
+from redcap_bridge.test_redcap.test_utils import (test_directory, initialize_test_dir)
+
+SERVER_CONFIG_YAML = (test_directory / 'testfiles' / 'TestProject' / 'project.json').resolve()
+
+test_file = (test_directory / 'testfiles' / 'elabConversion' / 'csvRecord.csv')
+
+
+def test_create_template(initialize_test_dir):
+    template_file = test_directory / 'testfiles' / 'elab_template.json'
+
+    res, http_stat_code = create_template(server_config_json=SERVER_CONFIG_YAML, template_file=template_file,
+                                          metadata=True)
+
+    assert http_stat_code == 200 or http_stat_code == 201
```

Comment:
@killianrochet Are you sure both `200` and `201` code for a successful creation of the template?

Reply:
Hello, yes i'm sure because 200 is used for the patch function of a template on the API and 201 is used for the POST function (creation) of a template on the API